### DISCUSSION TITLE
Owncast in local LAN

### DISCUSSION THREAD

**Discussion by hsc21** [owncast/owncast#1144] [General Owncast discussion]
Hi,

We try to run Owncast without Internet connection on a local server. 

But: 
The styles on the admin page are crapy 
The stream is not working. (Only the Logo is shown)

Ist there a way th get the stream runing in  local LAN without Internet connection of the Server?

**Comment by gabek** [comment]
What errors are you seeing in your browser console?

**Reply by hsc21** [reply]
![signal-2021-07-01-103415 (1)](https://user-images.githubusercontent.com/86737494/124099992-57d09300-da5e-11eb-8aae-7f76c9630cd2.png)
![signal-2021-07-01-103415](https://user-images.githubusercontent.com/86737494/124099996-599a5680-da5e-11eb-8663-fbdf52a0ee70.png)
On the admin page *:8080/admin the Layout is ugly
And the Play "Button" in the Stream is Missing 

Screenshots attached

**Reply by gabek** [reply]
I mean your browser console.  You'll need to provide some actual error messages so you can tell me what's wrong.  There's likely some 404s or Javascript errors or something.  Please open your browser's console and let me know what they are.

https://appuals.com/open-browser-console/

**Comment by hsc21** [comment]
we reinstalled Owncast, 
used the Port correct port along with the IP-Adress.

**now it works**

-> not a Owncast issue
Thanks for support
